### PR TITLE
add HF2_7K support

### PR DESCRIPTION
**Pull request by SamKChang**
add 'large' keyword to 'load_HF2' function to load full 7K HF2 dataset from QM_wavelet repo

### CODE REVIEW

File: cheml/datasets.py [hunk 1]
```diff
@@ -184,13 +186,16 @@ def load_qm7(path=None, align=False, only_planar=False, planarity_tol=.01):
     if align or only_planar:
         pca = PCA()
         keep_molecule = []
-        for molecule in qm7_bunch.R:
-            transformed = pca.fit_transform(molecule)
+        for positions, charges in zip(qm7_bunch.R, qm7_bunch.Z):
```
Comment (eickenberg) [inline cheml/datasets.py]: hmm this change should be merged, right? Could you lok whether your local version is rebased on current master?

**Comment by eickenberg** [comment]
awesome, that was missing!

Could you take a very quick look at my PR introducing load_HXn also?

File: cheml/datasets.py (hunk 1 above)
Comment (eickenberg) [inline cheml/datasets.py]: ah, it looks like you continued in the same branch as your previous PR, maybe?

Comment (SamKChang) [inline cheml/datasets.py]: I think so... Sorry, I am not very familiar with branching. I think I am still in the same PR. Should I make a new branch for this?

**Comment by eickenberg** [comment]
@SamKChang either reclone your repo or do `git reset --hard`

Then you will see that `git remote -v` gives you two repos under the name of `origin` pointing to `https://github.com/SamKChang/CheML.git`.

You should then do once and for all
`git remote add upstream https://github.com/CheML/CheML.git`
That way you will be able to keep up to date with the main repo.

from time to time execute:
```
git fetch upstream
git checkout master
git rebase upstream/master
git push origin master
```

to keep your own master branch in sync with the remote upstream one.
In order to add a new feature, opt for using branches:

```
git checkout -b new_feature_branch_name
```
code there, commit whatever you want.
Keep up with `upstream/master` by doing from time to time:

```
git fetch upstream
git rebase upstream/master
git push origin new_feature_branch_name
```

Do your PRs from that branch.

**Comment by SamKChang** [comment]
THANKS!

**Pull request closed without merging**